rotator_pxi/README.md

```diff
@@ -0,0 +1,60 @@
+# Software Environment of Rotator PXI Controller
+
+*This is the software environment of rotator PXI controller.*
```

Comment:
This will make text italic. Not sure if you want it, just remove it - it doesn't make any sense here. Use * to make things italic when needed.

Reply:
I agree that removing this line is probably best. It duplicates the line above it.

Reply:
Fixed in commit: ce7f486. Thanks!